compiler/AST/symbol.cpp

```diff
@@ -1598,6 +1598,16 @@ int FnSymbol::hasGenericFormals() const {
   bool hasGenericDefaults =  true;
   int  retval             =     0;
 
+  bool inInitRes = false;
```

Comment:
What does "res" mean in inInitRes? Maybe there's a better name.  Also alignment does not
appear to match the nearby locals if github is rendering correctly.